Sources/Testing/Testing.docc/known-issues.md

```diff
@@ -0,0 +1,38 @@
+# Known issues
+
+<!--
+This source file is part of the Swift.org open source project
+
+Copyright (c) 2023 Apple Inc. and the Swift project authors
+Licensed under Apache License v2.0 with Runtime Library Exception
+
+See https://swift.org/LICENSE.txt for license information
+See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+-->
+
+Call-out expected failures or warnings in tests.
```

Comment:
```suggestion
Highlight known issues when running tests.
```

Or "suppress" or "call out"—"call-out" is an old-timey noun, not a verb.

Avoid "expected failures" as "expect" is already a term of art for `#expect()`/`Expectation` and we consistently refer to problems in a test with the noun "issues" rather than "failures" (e.g. "a test that records an issue will fail".)

Reply:
That's on me then, I guess. :)